content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -68,47 +72,162 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
+   You can validate this behavior by using kubectl to inspect an existing service.
+
+```yaml
+kubectl get svc my-service -o yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: MyApp
+  name: my-service
+spec:
+  clusterIP: 10.0.197.123
+  clusterIPs:
+  - 10.0.197.123
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: MyApp
+  type: ClusterIP
+status:
+  loadBalancer: {}
+```
+
+   2. When dual-stack is enabled on a cluster, existing [headless Services](/docs/concepts/services-networking/service/#headless-services) with selectors are configured by the control plane to set `.spec.ipFamilyPolicy` to `SingleStack` and set `.spec.ipFamilies` to the address family of the first service cluster IP range (configured via the `--service-cluster-ip-range` flag to the kube-controller-manager) even though `.spec.ClusterIP` is set to `None`. 
 
-   * Kubenet forces IPv4,IPv6 positional reporting of IPs (--cluster-cidr)
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
 
+   You can validate this behavior by using kubectl to inspect an existing headless service with selectors.
+
+```yaml
+kubectl get svc my-service -o yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: MyApp
+  name: my-service
+spec:
+  clusterIP: None
+  clusterIPs:
+  - None
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: MyApp
+```
+
+#### Switching Services between single-stack and dual-stack
+
+Services can be changed from single-stack to dual-stack and from dual-stack to single-stack.
+
+   1. To change a Service from single-stack to dual-stack, change `.spec.ipFamilyPolicy` from `SingleStack` to `PreferDualStack` or `RequireDualStack` as desired. When you change this Service from single-stack to dual-stack, Kubernetes assigns the missing address family so that the Service now has IPv4 and IPv6 addresses.
+
+   Edit the Service specification updating the `.spec.ipFamilyPolicy` from `SingleStack` to `PreferDualStack`.
+
+Before:
+```yaml
+spec:
+  ipFamilyPolicy: SingleStack
+```
+After:
+```yaml
+spec:
+  ipFamilyPolicy: PreferDualStack
+```
+
+   2. To change a Service from dual-stack to single-stack, change `.spec.ipFamilyPolicy` from `PreferDualStack` or `RequireDualStack` to `SingleStack`. When you change this Service from dual-stack to single-stack, Kubernetes retains only the first element in the `.spec.ClusterIPs` array, and sets `.spec.ClusterIP` to that IP address and sets `.spec.ipFamilies` to the address family of `.spec.ClusterIPs`.
+
+### Headless Services without selector
+
+For [Headless Services without selectors](/docs/concepts/services-networking/service/#without-selectors) and without `.spec.ipFamilyPolicy` explicitly set, the `.spec.ipFamilyPolicy` field defaults to `RequireDualStack`.
+
+### Service Type LoadBalancer
```

Comment:
nit: Service type load balancer?

Reply:
As we're talking about a Service of this LoadBalancer type, we tried to keep the capitalization as it is in the other docs and code: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer